apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -5,11 +5,14 @@ import { type ValueOf } from "~/libs/types/types.js";
 
 type Properties = {
 	children: React.ReactNode;
+	className?: string;
```

Comment:
I was already told not to pass className to common components. If we need something specific, we should pass it as a separate prop